share/functions/fish_default_key_bindings.fish

```diff
@@ -13,7 +13,7 @@ function fish_default_key_bindings -d "Default (Emacs-like) key bindings for fis
     end
 
     # This is the default binding, i.e. the one used if no other binding matches
-    bind $argv "" self-insert
+    bind $argv \uE000 self-insert
```

Comment:
@krader1961 Yeah I kind of hesitated on using the private-use char also. Unfortunately under the current way the input stuff is set up there's no way to get around putting some "filler character" in as the self-insert binding. My original pull request tried to do away with this by having it self-insert when no match was found but I quickly ran into trouble with the vi bindings in normal mode in which regular input is disabled.

I can't think of a way to get around using a private key binding as the bind character unless we want to add in some specific method to determine whether we're in vi normal mode and if so prevent self-insert.